media/js/base/mozilla-fxa-form.js

```diff
@@ -34,22 +34,18 @@ if (typeof window.Mozilla === 'undefined') {
     FxaForm.getUTMParams = function() {
         var urlParams = new window._SearchParams().utmParams();
         var attributionData = Mozilla.UtmUrl.getAttributionData(urlParams);
+        var utmParams = {};
 
         if (attributionData) {
             var utms = ['utm_source', 'utm_campaign', 'utm_content', 'utm_term', 'utm_medium'];
             for (var i = 0; i < utms.length; i++) {
                 if (Object.prototype.hasOwnProperty.call(attributionData, utms[i])) {
-                    // check if input is available
-                    if (formElem.querySelector('[name="' + utms[i] + '"]')) {
-                        formElem.querySelector('[name="' + utms[i] + '"]').value = attributionData[utms[i]];
-                    } else {
-                        // create input if one is not present
-                        var input = FxaForm.createInput(utms[i], attributionData[utms[i]]);
-                        formElem.appendChild(input);
-                    }
+                    utmParams[i] = attributionData[utms[i]];
                 }
             }
         }
+
+        return utmParams;
```

Comment:
Nit: add  comment at the top of this function to describe what's now being returned e.g. `* @returns {Array} of utm parameters`